README.md

```diff
@@ -100,6 +102,26 @@ Adding the following to your workflow will create a new Sentry release and tell
         SENTRY_URL: https://sentry.example.com/
     ```
 
+## Releases
```

Comment:
The rich diff shows more clearly that I'm moving the Troubleshooting section down below and I'm making some minor changes to these sections
<img width="722" alt="image" src="https://user-images.githubusercontent.com/44410/234895466-6cfefe15-9a34-4634-9126-43c9029c6e60.png">